### ISSUE TITLE
error: YAML Exception

### ISSUE THREAD

**Issue by Takazudo** [encharm/Font-Awesome-SVG-PNG#21]
HI, CodeCharmLtd.

Thank you for the nice software.
Today, I got an error below.

```bash
$ font-awesome-svg-png --color red --sizes 32
Downloading latest icons.yml ...
Downloading latest fontawesome-webfont.svg ...

YAMLException: JS-YAML: end of the stream or a document separator is expected at line 8, column 25:
            background-color: #f1f1f1;
                            ^
```

Any ideas?

Thanks.

**Comment by jpdevries** [comment]
I just ran into this also after

```bash
npm install -g font-awesome-svg-png
brew install librsvg #required lib
```

```bash
font-awesome-svg-png --color red --sizes 128,256
Downloading latest icons.yml ...
Downloading latest fontawesome-webfont.svg ...

YAMLException: JS-YAML: end of the stream or a document separator is expected at line 8, column 25:
            background-color: #f1f1f1;
                            ^
```

**Comment by Rush** [comment]
Just published 1.1.2 - sorry for taking so long
